de.js

```diff
@@ -920,63 +920,63 @@ chall.name.dmg {0} Schaden zufügen {1}
 chall.name.bzk Erziele {0} Spielverderber-Abschüsse {1}
 chall.name.cc Erziele {0} hauchnahe Abschüsse {1}
 chall.name.rk Erziele {0} Vergeltungs-Abschüsse {1}
-event.doubleXP 2x Class XP enabled for this match
-ranked.join Joined Ranked Match. Abandoning a match will result in a temporary ban
-ranked.progress Game in Progress
-ranked.team Please join a Team
-ranked.switch Can't switch while in-game
-ranked.spectate Spectating Disabled in Ranked
-ranked.login Login to Spawn
+event.doubleXP 2x Klassen XP ein für dieses Match
+ranked.join Joined Ranked Spiel beigetreten. Ein ranked Match verlassen wird mit einem temporären Bann bestraft
+ranked.progress Spiel in Gange
+ranked.team Tritt einem Team bei
+ranked.switch Du kannst während der Runde deine Klasse nicht wechseln
+ranked.spectate Zuschauen ist im Ranked modus nicht gestattet
+ranked.login Logge dich ein um zu spawnen
```

Comment:
Melde dich an um zu spawnen
Login has to be replaced with Anmeldung everywhere for consistency